packages/node_modules/@webex/plugin-meetings/src/meetings/request.js

```diff
@@ -34,6 +34,15 @@ export default class MeetingRequest extends StatelessWebexPlugin {
     return this.webex.internal.services.fetchClientRegionInfo();
   }
 
+  /**
+     *  fetch login user infomation
+     * @returns {Promise<object>} geoHintInfo
+    */
+  fetchLoginUserInfomation() {
```

Comment:
`fetchLoginUserInfomation` can be changed to `fetchLoginUserInformation`, I feel this is a typo.

Reply:
aren't both the same one can you re check it once

Reply:
you're missing "r" in Information

Reply:
done